tests/commands/install_uninstall_test.py

```diff
@@ -726,6 +726,46 @@ def test_commit_msg_legacy(commit_msg_repo, tempdir_factory, store):
     assert second_line.startswith('Must have "Signed off by:"...')
 
 
+def test_post_commit_integration(tempdir_factory, store):
+    path = git_dir(tempdir_factory)
+    config = [
+        {
+            'repo': 'local',
+            'hooks': [{
+                'id': 'post-commit',
+                'name': 'Post commit',
+                'entry': 'touch post-commit.tmp',
+                'language': 'system',
+                'always_run': True,
+                'verbose': True,
+                'stages': ['post-commit'],
+            }],
+        },
+        {'repo': 'meta', 'hooks': [{'id': 'identity'}]},
+    ]
+    write_config(path, config)
+    with cwd(path):
+        cmd_output('git', 'add', '.')
+        git_commit()
+
+        assert not os.path.exists('post-commit.tmp')
+
+        open('some_file', 'a').close()
+        cmd_output('git', 'add', '.')
+        git_commit()
```

Comment:
use `_get_commit_output(...)`, otherwise this test pollutes the user's pre-commit cache

Reply:
I'm not sure if I understand this one correctly. You mean `_get_commit_output(tempdir_factory)` instead of `git-commit()`, right?

Reply:
yes, I'm not sure how else you could interpret this comment ? :)

Reply:
i was unsure if it's meant to be an addition or a replacement.